### PR TITLE
Clarify how to add event listener to element

### PR DESCRIPTION
**Pull request by katejeffreys**

### CODE REVIEW

**Comment by katejeffreys** [comment]
Fixes #2406

**Comment by katejeffreys** [comment]
@kevinpschaaf @arthurevans WDYT?

File: app/2.0/docs/devguide/events.md [hunk 1]
```diff
@@ -51,13 +51,16 @@ methods to add and remove event listeners imperatively.
 
 ### Listener on the custom element
 
-Listeners on a custom element can be set up in `ready()` using `this.addEventListener()`.
-The listener will be set up the first time the custom element is attached to the DOM.
+Listeners on a custom element can be set up using `this.addEventListener()`. Depending on
+the event you need to listen for, choose an appropriate point in a [custom element lifecycle callback](custom-elements#element-lifecycle)
+at which to add the listener.
+
+For example:
```
Comment (kevinpschaaf) [inline app/2.0/docs/devguide/events.md]: How about:
> For example, the following placement will ensure listeners are added once, before the element stamps its template and reacts to initial property values:

**Comment by katejeffreys** [comment]
@kevinpschaaf thanks for the suggestion, that's perfect - updated

**Pull request merged**